### ISSUE TITLE
Feature Request:  Insert only reference without bracket

### ISSUE THREAD

**Issue by danieltomasz** [chrisgrieser/alfred-bibtex-citation-picker#24]
### Feature Requested

Would be great to have a shortcut to add only reference wirthout bracket

For example I want to have complex citation [@authorx2022, @authory2022], currently I need to manually remove additional brackets when trying to insert the reference into existeing brackets `[[@authorx2022] @authory2022]`

### Checklist

- [X] The feature would be useful to more users than just me.

**Comment by chrisgrieser** [comment]
Good suggestion. Added in the latest release, triggered via `⌘⇧ + ↵`. For the specifics, [see the updated docs on citation actions](https://github.com/chrisgrieser/alfred-bibtex-citation-picker#citation-actions).

(You will to update to the latest release by manually downloading it at GitHub, since I had to remove the auto-update mechanism for the upcoming Alfred Gallery. Auto-updating will be available again via Alfred in 1-2 days when the workflow is published in the Gallery.)
